### PR TITLE
Reduce code redundancy

### PR DESCRIPTION
**Pull request by bchinmay-star**

### CODE REVIEW

File: app/src/main/java/com/example/varchasdream11/Fragments/ContestFragment.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                             Bundle savedInstanceState) {
+
+
+
+        //CRICKET ACTIVITY INTENT -------------------------->
+        View rootView = inflater.inflate(R.layout.fragment_contest, container, false);
+        Button button2 = (Button) rootView.findViewById(R.id.button2);
+        button2.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                Intent intent = new Intent(getActivity(), Cricket.class);
```
Comment (imraghav20) [inline app/src/main/java/com/example/varchasdream11/Fragments/ContestFragment.java:74]: Rename `Cricket.class` to `Sports.class`

Comment (imraghav20) [reply app/src/main/java/com/example/varchasdream11/Fragments/ContestFragment.java:74]: Make this change everywhere.

File: app/src/main/java/com/example/varchasdream11/Matches/Cricket.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            super(itemView);
+
+            matchTime = itemView.findViewById(R.id.matchTime);
+            team1Text = itemView.findViewById(R.id.team1Text);
+            team2Text = itemView.findViewById(R.id.team2Text);
+            team1Image = itemView.findViewById(R.id.team1Image);
+            team2Image = itemView.findViewById(R.id.team2Image);
+        }
+    }
+}
```
Comment (imraghav20) [inline app/src/main/java/com/example/varchasdream11/Matches/Cricket.java:105]: Rename this file as `Sports.java`

File: app/src/main/res/layout/activity_cricket.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    android:background="@drawable/gradient_background"
+    >
+
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/cricketRecyclerList"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+</RelativeLayout>
```
Comment (imraghav20) [inline app/src/main/res/layout/activity_cricket.xml:18]: Rename as `activity_sports.xml`

**Review by imraghav20 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_cricket.xml (hunk 3 above)
Comment (VortexExpansion) [reply app/src/main/res/layout/activity_cricket.xml:18]: Done all the above changes in PR#8

File: app/src/main/java/com/example/varchasdream11/Fragments/ContestFragment.java (hunk 1 above)
Comment (bchinmay-star) [reply app/src/main/java/com/example/varchasdream11/Fragments/ContestFragment.java:74]: Done

File: app/src/main/java/com/example/varchasdream11/Matches/Cricket.java (hunk 2 above)
Comment (bchinmay-star) [reply app/src/main/java/com/example/varchasdream11/Matches/Cricket.java:105]: Done

**Pull request merged**